### PR TITLE
Create remote.md

### PR DESCRIPTION
**Pull request by yangc95**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@yangc95 Looks good to me.

File: content/git/concepts/remote/remote.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Remote'
```
Comment (caupolicandiaz) [inline content/git/concepts/remote/remote.md:2]: I think we might want to change the title and the path to the plural which is our usual formatting

File: content/git/concepts/remote/remote.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Description: 'Remotes allow keep tabs on the remote repositories (the repos on GitHub).'
```
Comment (caupolicandiaz) [inline content/git/concepts/remote/remote.md:3]:
```suggestion
Description: 'Repositories that are not local.'
```

File: content/git/concepts/remote/remote.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-git'
+  - 'learn-the-command-line'
+---
+
+Git **remotes** keep tabs on the remote repositories (the repos on GitHub). The `origin` remote refers to the default remote repo while the `upstream` remote refers to the remote repo that is being tracked for changes and contributions.
```
Comment (caupolicandiaz) [inline content/git/concepts/remote/remote.md:15]:
```suggestion
Git **remotes** are copies of repositories that are not stored locally. Working with a remote often involves two copies or references: the forked repository and the base or home repository. By convention these are named the `origin` remote which refers to the default, forked repo. And the `upstream` remote, which refers to the remote repo that is being tracked for changes and contributions.
```

**Review by caupolicandiaz — APPROVED** [review]
@yangc95 Just added some suggestions to expand the definition a bit. Otherwise looks good

**Pull request merged**